test/functional/http/post/4.multisignature.advanced.js

```diff
@@ -33,11 +33,15 @@ describe('POST /api/transactions (type 4) register multisignature', () => {
 		offline_signed_empty_signatures: new Scenarios.Multisig(),
 		offline_signed_without_ready: new Scenarios.Multisig(),
 		offline_signed_with_ready_false: new Scenarios.Multisig(),
+		unsigned_with_ready_true: new Scenarios.Multisig(),
 		offline_signed_with_ready_true: new Scenarios.Multisig(),
 		duplicated_signature: new Scenarios.Multisig(),
 		extra_signature: new Scenarios.Multisig(),
 		unknown_signature: new Scenarios.Multisig(),
 		requesterPublicKey: new Scenarios.Multisig(),
+		all_signatures_ready_false: new Scenarios.Multisig(),
```

Comment:
These cases are in conflict with the ones at the beginning starting by `offline_signed*`.

Reply:
They do differ in the addition of the ready property for testing the cases when it is present in the transaction.

Reply:
@pablitovicente `offline_signed_with_ready_false` & `all_signatures_ready_false` are pretty much the same. Please, follow the notation to avoid misunderstandings.
`no_signatures_ready_true` is equal to `unsigned_with_ready_true`
`some_signatures_ready_true `should be o `offline_partly_signed_with_ready_true` or `partly_offline_signed_with_ready_true`

Reply:
Improved the names in two cases and removed one that was unused.